modules/cardinal_service_profile_helper/templates/components/opportunity-grid/opportunity_grid.html.twig

```diff
@@ -16,7 +16,16 @@
 	{% if program is not empty or type_list is not empty %}
     {% set service_theme = service_theme|merge({'#markup': service_theme['#markup'] ~ ', '}) %}
 	{% endif %}
+{% endif %}
+
+{# Add "comma" between time_commitment is not empty and service_theme is. #}
+{% if time_commitment is not empty and service_theme is empty %}
+	{% set time_commitment = time_commitment|merge({'#markup': time_commitment['#markup'] ~ ', '}) %}
+{% endif %}
 
+{# Add "comma" between time_commitment is not empty and service_theme is. #}
+{% if time_commitment is empty and service_theme is not empty %}
+  {% set service_theme = service_theme|merge({'#markup': service_theme['#markup'] ~ ', '}) %}
```

Comment:
Real close here.

1. Both of these conditions should also check if one of Program or Type List is not empty -- if both are empty there shouldn't be a comma added.
2. Update the comments above the conditions to properly reflect the conditions.
3. Update the PR title with the ticket number.